<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Port">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Порт:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Chat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>471</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="Vvod_Nomer_Porta">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>670</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="go_fly">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>500</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Вжух 
Отправить</string>
    </property>
   </widget>
   <widget class="QLabel" name="Start_text">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>331</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#5500ff;&quot;&gt;Введите ваш никнэйм:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="go">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Подтвердить</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Vvod">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Vvedite_vash_message">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите ваше сообщение:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name_Set">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Set_Port">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>730</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Установить порт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_online">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Пользователи онлайн:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ofline">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Для ЛС:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_online">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>231</width>
      <height>721</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="list_for_IP">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>231</width>
      <height>721</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
